packages/assistant-stream/src/core/serialization/data-stream/DataStream.ts

```
@@ -248,25 +233,18 @@ export class DataStreamDecoder extends PipeableTransformStream<
-                );
-              toolCallController.unstable_setArtifact(artifact);
-              break;
-            }
-
             case DataStreamStreamChunkType.ToolCallResult: {
-              const { toolCallId, result } = value;
+              const { toolCallId, artifact, result, isError } = value;
               const toolCallController = toolCallControllers.get(toolCallId);
               if (!toolCallController)
                 throw new Error(
                   "Encountered tool call result with unknown id: " + toolCallId,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)